docs/tutorials/gcp_gpu.md

```diff
@@ -0,0 +1,195 @@
+        the pattern `cuda-XX.Y`.  The output of the command should resemble the
+        following:
+
+        ```shell
+        bin cuda cuda-11 cuda-11.4 etc games include lib man sbin share src
+        ```
+
+        In this case, the directory is `cuda-11.4`.
+    2.  Add the CUDA toolkit path to your environment by appending the following
+        line to your `~/.bashrc` file.  Replace `[DIR]` with the CUDA directory that
+        you discovered in the previous step.
+
+        ```shell
+        echo "export PATH=/usr/local/[DIR]/bin${PATH:+:${PATH}}" >> ~/.bashrc
+        ```
+
+    3.  Run `source ~/.bashrc` to activate the new environment search path
+
+## 4. Install build tools
+
+Run the following command to install the Cmake tool and the pip tool. This step
+might take a few minutes to complete.
+
+```shell
+sudo apt install cmake && sudo apt install pip
+```
+
+
+## 5. Create a GPU-enabled version of qsim
+
+1.  Run the following command to clone the qsim repository.
+
+    ```shell
+    git clone https://github.com/quantumlib/qsim.git
+    ```
+
+2.  Run `cd qsim`to change your working directory to qsim.
+3.  Run `make`to compile qsim. When CUDA toolkit is installed during a qsim
+    compilation, make builds the GPU version automatically.
+4.  Run `pip install .` to install your local version of qsimcirq.
+5.  Run `python3 -c "import qsimcirq; print(qsimcirq.qsim_gpu)"`to verify your
+    installation. If the installation completed successfully, the output from
+    the command should resemble the following:
+
+    ```shell
+    <module 'qsimcirq.qsim_cuda' from '/home/user_org_com/qsim/qsimcirq/qsim_cuda.cpython-38-x86_64-linux-gnu.so'>
+    ```
+
+
+## 6. Verify your installation
+
+You can use the code below to verify that qsim uses your GPU. You can paste the
+code directly into the REPL, or paste the code in a file and ask the Python
+interpreter to run it. If you paste the code into a file, use the following
+command in the interpreter to run it. Replace `[FILE]` with the name of the file
+that you create.
+
+```
+>>> exec(open("[FILE]").read())
+```
+
+In the `qsim` directory, run `python3` to start the Python REPL. Run the
+following code.
+
+```
+# Import Cirq and qsim
+import cirq
+import qsimcirq
+
+# Instantiate qubits and create a circuit
+q0, q1 = cirq.LineQubit.range(2)
+circuit = cirq.Circuit(cirq.H(q0), cirq.CX(q0, q1))
+
+# Instantiate a simulator that uses the GPU
+gpu_options = qsimcirq.QSimOptions(use_gpu=True)
+qsim_simulator = qsimcirq.QSimSimulator(qsim_options=gpu_options)
+
+# Run the simulation
+print("Running simulation for the following circuit:")
+print(circuit)
+
+qsim_results = qsim_simulator.compute_amplitudes(
+    circuit, bitstrings=[0b00, 0b01])
+
+print("qsim results:")
+print(qsim_results)
+```
+
+After a moment, you should see a result that looks similar to the following.
+
+```
+[(0.7071067690849304+0j), 0j]
+```
+
+## Clean up
```

Comment:
Is this section supposed to be in the "before you begin" doc?

If not, there's duplication between it and the `Next steps` section below.

Reply:
Fixed.